README.md

```diff
@@ -26,6 +26,21 @@ $ export ETH_RPC_URL=<Mainnet URL>
 $ make test
 ```
 
+### Test (Forge without Optimizations)
+
+#### Prerequisites
+1. [Install](https://www.rust-lang.org/tools/install) Rust.
+2. [Install](https://github.com/gakonst/foundry#forge) Forge.
+3. Add the Cargo binaries directory (e.g. `~/.cargo/bin`) to your PATH env var.
```

Comment:
I think this point is not required as when installing `foundry` via either readme specs or `forgeup` it should add the following in `.bashrc`:
`. "$HOME/.cargo/env"`
that should add the binary directory to the PATH.

Reply:
applied in fbf2f53.